src/roles.json

```diff
@@ -1497,11 +1497,37 @@
     "firstNight": 54,
     "firstNightReminder": "Wake any Village Idiot. They point to a player. Give a thumbs up or a thumbs down. Put that Village Idiot to sleep. Repeat until all Village Idiots have acted.",
     "otherNight": 65,
-    "otherNightReminder": "wake any Village Idiot. They point to a player. Give a thumbs up or a thumbs down. Put that Village Idiot to sleep. Repeat until all Village Idiots have acted.",
+    "otherNightReminder": "Wake any Village Idiot. They point to a player. Give a thumbs up or a thumbs down. Put that Village Idiot to sleep. Repeat until all Village Idiots have acted.",
     "reminders": ["Drunk"],
     "setup": true,
     "ability": "Each night, choose a player: you learn their alignment. [There can be 1-3 Village Idiots. If 2+ are in play, 1 Village Idiot is drunk]"
   },
+  {
+    "id": "banshee",
+    "name": "Banshee",
+    "edition": "",
+    "team": "townsfolk",
+    "firstNight": 0,
+    "firstNightReminder": "",
+    "otherNight": 42,
+    "otherNightReminder": "If the Demon kills the Banshee, announce (at night) “The Banshee has awoken”; from now, the Banshee may nominate twice per day, and may raise one or two hands when voting. When counting the votes, count each hand as a vote.",
```

Comment:
I'm guessing we're just gonna use the normal vote and additional little hand we created to support multiple hands up by someone?

Reply:
> I'm guessing we're just gonna use the normal vote and additional little hand we created to support multiple hands up by someone?

Yeah, it's a neat & easy solution for this character and our townsquare